translations/localify/ui.json

```diff
@@ -2287,5 +2287,40 @@
     "選択数": "Selected",
     "期限が切れたプレゼントは<color=#ff911c>自動的に削除</color>されます\n一度に受け取れるプレゼントは<color=#ff911c>100件</color>までです": "Expired presents will be <color=#ff911c>auto-deleted</color>\nYou can only receive <color=#ff911c>100 gifts</color> at a time",
     "受け取れるプレゼントはありません": "No presents",
-    "ピースを変換できます。\n変換で獲得したクローバーは\nアイテムと交換できますよ。": "Pieces can be converted. \nThe clovers obtained by conversion \ncan be used to buy items."
-}
+    "ピースを変換できます。\n変換で獲得したクローバーは\nアイテムと交換できますよ。": "Pieces can be converted. \nThe clovers obtained by conversion \ncan be used to buy items.",
+    "このイベントを再生しますか？": "Do you want to replay this event?",
+    "分岐は<color=#ff9900>ランダムで再生</color>されます": "Branches will be <color=#FF9900>replayed randomly</color>",
+    "トーク": "Talk",
+    "アプリ終了確認": "Application exit confirmation",
+    "アプリを終了してもよろしいですか？": "Are you sure you want to exit the application?",
+    "育": "Training",
+    "成": "<empty>",
+    "イ": "<empty>",
+    "ベン": "Event",
+    "ト": "<empty>",
+    "ギ": "Gallery",
+    "ャ": "<empty>",
+    "ラ": "<empty>",
+    "リ": "<empty>",
+    "ー": "<empty>",
+    "あら、\n今日はどんなご用ですか？": "Oh, \nwhat can I do for you today?",
+    "あら、今日はどんなご用ですか？": "Oh, what can I do for you today?",
+    "名鑑レベルが上がると\n報酬が手に入ります。\nがんばりましょうね！": "You will get rewards \nas your index level goes up. \nKeep up the good work!",
+    "名鑑レベルが上がると報酬が手に入ります。がんばりましょうね！": "You will get rewards as your index level \ngoes up. Keep up the good work!",
+    "歴代評価点を更新し、\n名鑑レベルを上げて\nいきましょう！": "Let's update your past \nrating scores and raise \nyour index level!",
+    "歴代評価点を更新し、名鑑レベルを上げていきましょう！": "Let's update your past rating scores \nand raise your index level!",
```

Comment:
Part of mdb. Check the `character_system_text` entry in `src/mdb/index.json` and add as needed.
In this case Tazuna's id is 9001 so she can be added to the files dict like so: `"tazuna": 9001`

This will create a `tazuna.json` file with all her system text under `translations/mdb/character_system_text` where you can do the translations.
Currently this results in suboptimal formatting but I am working on incorporating these files in TLG.
This way the text will be translated on mobile (and for anyone not using TLG) too, albeit with said formatting issue. It's also easier to deal with and won't require dumping (and finding!) all the text from in-game once my changes are merged.